aea/crypto/ethereum.py

```diff
@@ -506,6 +506,15 @@ def try_estimate_gas(self, tx: Dict[str, Any]) -> Dict[str, Any]:
             _default_logger.debug("Error when trying to estimate gas: {}".format(e))
         return tx
 
+    @classmethod
+    def is_valid_address(cls, address: Address) -> bool:
```

Comment:
add validation to ledger methods to check the address on transaction construction or balance check?

Reply:
will do in a separate pr